src/b00_breeze.modelLibrary.ko.js

```diff
@@ -91,13 +91,15 @@
 
     var stype = proto.entityType || proto.complexType;
     var es5Descriptors = {};
+    vor found = false;
     stype.getProperties().forEach(function (prop) {
       var propDescr = getES5PropDescriptor(proto, prop.name);
       if (propDescr) {
         es5Descriptors[prop.name] = propDescr;
+        found=true;
       }
     })
-    if (!__isEmpty(es5Descriptors)) {
```

Comment:
But you are correct about the error. The author intended `core.isEmpty` and something got lost in translation.
I prefer your answer but I'll stick with the author's intentions and use `core.isEmpty` instead.